public/docs/ganache/using.md

```diff
@@ -0,0 +1,64 @@
+# Working with Ganache
+
+[Ganache](/ganache) is an application that generates a personal blockchain you can use to run tests, execute commands, deploy contracts, and inspect state.
+
+![Ganache logo](/docs/ganache/images/ganache-logo.png)
+
+It is available in both a graphical format and as a terminal application (which was previously known as the TestRPC). Ganache is available for Windows, macOS, and Linux.
+
+## Installation
+
+### Ganache
+
+Installation of Ganache is only a few steps. [Download](https://github.com/trufflesuite/ganache/releases) the appropriate version for your OS:
```

Comment:
When I read this as a "few steps", it actually sounded harder than it is. I'd almost remove the step count and just call it "easy": i.e., 

> Installing Ganache is easy. [Download](https://github.com/trufflesuite/ganache/releases) the appropriate version for your OS then double click on the file:

Reply:
Fair point, though I try to avoid using the words "easy", "simple" etc. to describe an application's functions.

I'll just take that whole sentence out, as they'll see below just how little there is in the section!

Also, I would want to keep the double-click below the file name, as that more directly reflects the order of operations. Also keeps the section from ending on a bulleted list.